app/helpers/pipelines_helper.rb

```diff
@@ -15,6 +15,18 @@ def definition_edit_text(definition, type)
     "Edit #{type}"
   end
 
+  def definition_delete_text(definition, type)
+    # rubocop:disable Rails/OutputSafety
+    if definition.shared?
+      "<p>This will <strong>NOT</strong> delete the #{type} definition '#{definition.name}'
+          as it is shared with another pipeline.</p>".html_safe
+    else
+      "<p>This <strong>WILL</strong> delete the #{type} definition '#{definition.name}
+          as it is <strong>NOT</strong> shared with another pipeline.</p>".html_safe
+    end
+    # rubocop:enable Rails/OutputSafety
+  end
```

Comment:
I am not feeling comfortable with this html_safe as `definition.name` is user inputs...

Could you use `tag.p do` and `tag.strong` etc. to avoid the html_safe? not sure if it will help.

Reply:
Fair enough, I will drop the strong as it's not really critical and use a content tag :)